.ci/azure/linux.yml

```diff
@@ -181,7 +181,9 @@ jobs:
   - script: ls -alR $(REPO_DIR)/bin/
     displayName: 'List bin files'
 
-  - script: cmake -DCMAKE_INSTALL_PREFIX=$(INSTALL_DIR) -P cmake_install.cmake
+  - script: |
+      cmake -DCMAKE_INSTALL_PREFIX=$(INSTALL_DIR) -P cmake_install.cmake
+      python3 -m pip install $(BUILD_DIR)/wheels/*
```

Comment:
should not we use `--no-deps` as here https://github.com/openvinotoolkit/openvino_contrib/commit/13ed1c85cb1c48355e7d9f0a8ffe895daf48925f ?

CC @dkurt

Reply:
Without --no-deps pip tries to install unpiblished openvino runtime package